src/components/UI/InheritDot.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="inheritance-holder">
+  <div class="inheritance-holder" :class="{ 'inheritance-holder-left': colorPicker }">
```

Comment:
I'd rather have something like this. This way, the color field component will be responsible in dictating how the dot will be positioned, no the dot component being aware that "color picker" means "left aligned".

```suggestion
  <div class="inheritance-holder" :class="{ 'inheritance-holder-left': position === 'left' }">
```

Reply:
fixed